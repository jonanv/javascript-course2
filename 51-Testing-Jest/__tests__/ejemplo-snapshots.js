// Ejemplo snapshots
const cliente = {
    nombre: 'Juan',
    balace: 500,
    tipo: 'Premium'
}

describe('Testing al cliente', () => {
    test('Es Juan Pablo', () => {
        expect(
            cliente
        ).toMatchSnapshot();
    });
});