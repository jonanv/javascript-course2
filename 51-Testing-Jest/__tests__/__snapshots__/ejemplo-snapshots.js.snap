// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Testing al cliente Es Juan Pablo 1`] = `
{
  "balace": 500,
  "nombre": "Juan",
  "tipo": "Premium",
}
`;
